frontend/src/components/plugins/utils/utils.jsx

```diff
@@ -314,41 +309,46 @@ export function OrganizationPluginStateToggle({
   type,
   refetch,
 }) {
+  const user = useAuthStore(React.useCallback((s) => s.user, []));
   const {
     isUserOwner,
-    organization,
     noOrg,
     fetchAll: fetchAllOrganizations,
+    isUserAdmin,
   } = useOrganizationStore(
     React.useCallback(
       (state) => ({
         fetchAll: state.fetchAll,
         isUserOwner: state.isUserOwner,
-        organization: state.organization,
         noOrg: state.noOrg,
+        isUserAdmin: state.isUserAdmin,
       }),
       [],
     ),
   );
-  let title = disabled
-    ? `Enable ${pluginName} for organization`
-    : `Disable ${pluginName} for organization`;
-  if (!isUserOwner) {
-    title = `You're not an owner of your organization - ${organization.name}`;
-  }
-  let baseUrl = "";
-  if (type === pluginType.ANALYZER) {
-    baseUrl = ANALYZERS_CONFIG_URI;
-  } else if (type === pluginType.CONNECTOR) {
-    baseUrl = CONNECTORS_CONFIG_URI;
-  } else if (type === pluginType.VISUALIZER) {
-    baseUrl = VISUALIZERS_CONFIG_URI;
+  const { enablePluginInOrg, disabledPluginInOrg } =
+    usePluginConfigurationStore(
+      React.useCallback(
+        (state) => ({
+          enablePluginInOrg: state.enablePluginInOrg,
+          disabledPluginInOrg: state.disabledPluginInOrg,
+        }),
+        [],
+      ),
+    );
+  let title = `${
+    disabled ? "Enable" : "Disable"
+  } ${pluginName} for organization`;
+  if (!isUserOwner && !isUserAdmin(user.username)) {
+    title = `${pluginName} is ${
+      disabled ? "disabled" : "enabled"
+    } for the organization`;
   }
 
   const onClick = async () => {
-    if (isUserOwner) {
-      if (disabled) await axios.delete(`${baseUrl}/${pluginName}/organization`);
-      else await axios.post(`${baseUrl}/${pluginName}/organization`);
+    if (isUserOwner || isUserAdmin(user.username)) {
```

Comment:
I think with the update of the certego-saas the owner also is and admin so we don't need this ||